rero_ils/modules/fees/api.py

```diff
@@ -118,8 +118,8 @@ def create_fee_from_notification(cls, notification):
                 holding_circulation_category_pid
             )
             data['amount'] = cipo.get('overdue_amount')
-            data['currency'] = current_app.config\
-                .get('RERO_ILS_DEFAULT_CURRENCY')
+            data['currency'] = notification.organisation.\
+                get('default_currency')
```

Comment:
Do we need a default value if we do not have a `default_currency`key in the organisation?

Reply:
As `default_currency` should be mandatory into json_schema, no default value are required here (see next review comment)